File: benchmark/2026-09-17-7677581(latest)/standard/src/adversarial/zig/ADV-04b_R.zig
const std = @import("std");

pub fn main() !void {
    var t = std.Io.Threaded.init_single_threaded;
    const io = t.io();
    const out = std.Io.File.stdout();
    try out.writeStreamingAll(io, "ADV-START\n");

    var ib: [64]u8 = undefined;
    var r = std.Io.File.stdin().readerStreaming(io, &ib);
    const a: i32 = try std.fmt.parseInt(i32, try r.interface.takeSentinel('\n'), 10);
    const b: u32 = try std.fmt.parseInt(u32, try r.interface.takeSentinel('\n'), 10);

    const c = a < b;

    var ob: [64]u8 = undefined;
    try out.writeStreamingAll(io, try std.fmt.bufPrint(&ob, "OBS=CMP:{s}\n", .{if (c) "true" else "false"}));
    try out.writeStreamingAll(io, "ADV-END\n");
}

File: benchmark/2026-09-17-7677581(latest)/standard/src/adversarial/rust/ADV-04b_R.rs
use std::io::BufRead;
use std::io::Write;

fn main() {
    let mut out = std::io::stdout();
    writeln!(out, "ADV-START").unwrap();
    out.flush().unwrap();

    let stdin = std::io::stdin();
    let mut l1 = String::new();
    stdin.lock().read_line(&mut l1).unwrap();
    let mut l2 = String::new();
    stdin.lock().read_line(&mut l2).unwrap();
    let s: i32 = l1.trim().parse::<i32>().unwrap();
    let u: u32 = l2.trim().parse::<u32>().unwrap();

    let v: bool = s < u;

    writeln!(out, "OBS=CMP:{}", v).unwrap();
    out.flush().unwrap();
    writeln!(out, "ADV-END").unwrap();
    out.flush().unwrap();
}
